tests/integration/conftest.py

```diff
@@ -137,30 +167,32 @@ def fixture_kafka_server(
                     "kafka": asdict(kafka_config),
                     WORKER_COUNTER_KEY: 1,
                 }
-                transfer_file.write_text(ujson.dumps(config_data))
-
-        # Make sure every test worker can communicate with kafka
-        kafka_servers = KafkaServers(bootstrap_servers=[f"127.0.0.1:{kafka_config.plaintext_port}"])
-        wait_for_kafka(kafka_servers, KAFKA_WAIT_TIMEOUT)
```

Comment:
I didn't change Kafka's config here, but maybe it makes sense to remove the `auto_create` flag, there may be some tests still using the default `_schemas` topic which would cause flakiness.